src/DIRAC/ConfigurationSystem/private/Modificator.py

```diff
@@ -239,6 +242,13 @@ def __str__(self):
         return str(self.cfgData)
 
     def commit(self):
+        retOpt = gConfigurationData.extractOptionFromCFG(
+            "/Systems/Configuration/Services/Server/VerifyDiracXSyncOnCommit"
```

Comment:
The name is alright but here you should use 

```suggestion
        retOpt = self.getValue("/Systems/Configuration/Services/Server/VerifyDiracXSyncOnCommit")
```